web/templates/fedselect.frag

```diff
@@ -75,6 +75,7 @@
   </optgroup>
   <optgroup label="Finland">
     <option value="FPO">FPO - Finland Powerlifting Organization [IPA] (Incomplete)</option>
+    <option value="SVNL">SVNL - Suomen Voimanostoliitto [IPF]</option>
```

Comment:
I'm not sure, but I think it should be `Suomen Voimanostoliitto ry` based on their usage.

They should also be added to the list of "drug-tested" federations at the top of this file, and there should also be an "All Finland" option added.